www/runtest.php

```diff
@@ -538,7 +538,19 @@ function DealWithMagicQuotes(&$arr) {
                 $test['customMetrics'][$metric] = base64_encode($code);
               }
             }
-            
+
+            if (array_key_exists('heroElements', $_REQUEST)) {
+              $heroElements = json_decode($_REQUEST['heroElements']);
+              if (is_object($heroElements)) {
+                $heroElements = array_filter((array) $heroElements, function($selector) {
+                  // Hero elements should be an array of selector strings; filter
+                  // out invalid values.
+                  return is_string($selector) && strlen($selector);
+                });
+                $test['heroElements'] = json_encode($heroElements, JSON_FORCE_OBJECT);

```

Comment:
I'm not 100% sure the ini file format of test jobs will like a json object.  You will probably need to base64_encode/decode like we do for ['injectScript'].